src/app-layout/visual-refresh-toolbar/index.tsx

```diff
@@ -258,50 +276,60 @@ const AppLayoutVisualRefreshToolbar = React.forwardRef(
     };
 
     return (
-      <SkeletonLayout
-        style={{
-          [globalVars.stickyVerticalTopOffset]: `${verticalOffsets.header}px`,
-          [globalVars.stickyVerticalBottomOffset]: `${placement.insetBlockEnd}px`,
-          paddingBlockEnd: splitPanelOpen ? splitPanelReportedSize : '',
-        }}
-        toolbar={hasToolbar && <AppLayoutToolbar appLayoutInternals={appLayoutInternals} />}
-        notifications={
-          notifications && (
-            <AppLayoutNotifications appLayoutInternals={appLayoutInternals}>{notifications}</AppLayoutNotifications>
-          )
-        }
-        contentHeader={contentHeader}
-        content={content}
-        navigation={resolvedNavigation && <AppLayoutNavigation appLayoutInternals={appLayoutInternals} />}
-        navigationOpen={navigationOpen}
-        navigationWidth={navigationWidth}
-        tools={activeDrawer && <AppLayoutDrawer appLayoutInternals={appLayoutInternals} />}
-        toolsOpen={!!activeDrawer}
-        toolsWidth={activeDrawerSize}
-        sideSplitPanel={
-          splitPanelPosition === 'side' &&
-          splitPanel && (
-            <AppLayoutSplitPanelSide appLayoutInternals={appLayoutInternals} splitPanelInternals={splitPanelInternals}>
-              {splitPanel}
-            </AppLayoutSplitPanelSide>
-          )
-        }
-        bottomSplitPanel={
-          splitPanelPosition === 'bottom' && (
-            <AppLayoutSplitPanelBottom
-              appLayoutInternals={appLayoutInternals}
-              splitPanelInternals={splitPanelInternals}
-            >
-              {splitPanel}
-            </AppLayoutSplitPanelBottom>
-          )
-        }
-        splitPanelOpen={splitPanelOpen}
-        placement={placement}
-        contentType={contentType}
-        maxContentWidth={maxContentWidth}
-        disableContentPaddings={disableContentPaddings}
-      />
+      <>
+        {/* Rendering a hidden copy of breadcrumbs to trigger their deduplication */}
+        {!hasToolbar && breadcrumbs ? <ScreenreaderOnly>{breadcrumbs}</ScreenreaderOnly> : null}
+        <SkeletonLayout
+          style={{
+            [globalVars.stickyVerticalTopOffset]: `${verticalOffsets.header}px`,
+            [globalVars.stickyVerticalBottomOffset]: `${placement.insetBlockEnd}px`,
+            paddingBlockEnd: splitPanelOpen ? splitPanelReportedSize : '',
+          }}
+          toolbar={
+            hasToolbar && <AppLayoutToolbar appLayoutInternals={appLayoutInternals} toolbarProps={toolbarProps} />
+          }
+          notifications={
+            notifications && (
+              <AppLayoutNotifications appLayoutInternals={appLayoutInternals}>{notifications}</AppLayoutNotifications>
+            )
+          }
+          contentHeader={contentHeader}
+          // delay rendering the content until registration of this instance is complete
+          content={registered ? content : null}
```

Comment:
This delay is only for a single render loop, does not cause any observable rendering issues